tests/test_meltingpot.py

```diff
@@ -0,0 +1,57 @@
+"""Tests the functionality of the MeltingPotCompatibility wrapper on meltingpot substrates."""
```

Comment:
The tests look good to me, but perhaps @pseudo-rnd-thoughts may want to have a second look.